README.md

```diff
@@ -28,12 +27,11 @@ By the end of this workshop you will understand and have experience with the fol
 ## Before we start
 Before we'll dive into the workshop there are some important things you should know when working w/ Velo.
 * Monitoring and Debugging
-    * Backend logs can be viewed in the [Site Events (aka logs)](https://manage.wix.com/account/site-selector/?actionUrl=https%3A%2F%2Fmanage.wix.com%2Fdashboard%2F%7BmetaSiteId%7D%2Fsettings%2Fmonitoring%2Fsite-events&title=Site+Events&primaryButtonText=Select) section in the business manager's developer tools menu.
-    * Debugging client code can be done via the browser's developer tools. Look in the console for the file name of the page you want to debug
+    * Backend logs as well as client logs can be viewed in the [Site Events (aka logs)](https://manage.wix.com/account/site-selector/?actionUrl=https%3A%2F%2Fmanage.wix.com%2Fdashboard%2F%7BmetaSiteId%7D%2Fsettings%2Fmonitoring%2Fsite-events&title=Site+Events&primaryButtonText=Select) section in the business manager's developer tools menu. Note that you can't browse logs history, you can only see realtime logs, so the logs screen needs to be open during the session (open it in another tab in your browser).
```

Comment:
Good note about that the tab should be open. but you didn't explain about debugging client code in the browser..

Reply:
This is intentional, as I think its easier to debug both (client & backend) via Site Events (mostly because its in the same place), and not via dev tools.. Am I missing something special about debugging in dev tools that you tend to favor?